build.sbt

```diff
@@ -44,7 +44,7 @@ def module(proj: Project) = proj.
   settings(commonSettings, tutTargetDirectory := baseDirectory.value)
 
 lazy val akka = module(project) in file("modules/akka")
-lazy val cats = module(project) in file("modules/cats")
+lazy val cats = (module(project) in file("modules/cats"))
```

Comment:
Nitpick: why this change? It appears to be unnecessary and inconsistent with the surrounding code.

Reply:
Yeah, it's a leftover from when I was removing a 2.10 build. I'm rolling it back